### ISSUE TITLE
Local & Production Issues

### ISSUE THREAD

**Issue by AugmentBLU** [OFFLINE-GmbH/oc-cors-plugin#3]
Hello,

I have some font files hosted on Rackspace CDN and I'm getting the CORS warning so I found your plugin. 

Locally it doesn't seem to work, I still get the blocked warning and fonts are being pulled through. 

On production, I can't even sent any origins due to an error. When I click `Add new item`

> "" on line 882 of /home/forge/default/vendor/laravel/framework/src/Illuminate/Foundation/Application.php

Any idea what could be the issue with either?

Thanks

I just noticed that when I go to CMS and click on a page to update the same error on line 882 appears.

**Comment by tobias-kuendig** [comment]
What code is on line 882? This looks like a problem with your setup not the plugin. 

Concerning the CORS problem: Since the fonts are hosted on Rackspace you'll have to add your CORS headers there, not on your server/CMS.

**Comment by AugmentBLU** [comment]
Yeah you're right about the Rackspace settings - I did actually have these on but still had the issue. I realised some fonts had a version parameter on them ?v= and it seemed to mess with things. That side works now.

For the 882 error - this isn't something I've looked in to as yet, just wondered if you knew what it was or related to the plugin. I have my site deployed on Laravel Forge, that's the only difference between my local setup and production. I will dig deeper and see why it's not working.
